xen/arch/arm/coproc/coproc.c

```diff
@@ -183,9 +183,19 @@ static int coproc_attach_to_domain(struct domain *d,
         goto out;
     }
 
+    ret = iommu_assign_coproc(d, coproc->dev);
```

Comment:
No. Good question. I assume that IOMMU is mandatory for coproc to work at least in current our setup and taking into the account our limited experience in subject.